website/docs/r/alert_configuration.html.markdown

```diff
@@ -260,10 +260,14 @@ List of notifications to send when an alert condition is detected.
     - `USER`
     - `VICTOR_OPS`
     - `WEBHOOK`
+    - `MICROSOFT_TEAMS`
 
 * `username` - Name of the Atlas user to which to send notifications. Only a user in the project that owns the alert configuration is allowed here. Required for the `USER` notifications type.
 * `victor_ops_api_key` - VictorOps API key. Required for the `VICTOR_OPS` notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.
 * `victor_ops_routing_key` - VictorOps routing key. Optional for the `VICTOR_OPS` notifications type. If the key later becomes invalid, Atlas sends an email to the project owner and eventually removes the key.
+* `webhook_secret` - Authentication secret for the `WEBHOOK` notifications type.
```

Comment:
@martinstibbe is this going to be another potentially redacted value we need to tell users to ignore?  Or does this still get returned as the actual secret?   Also a minor change to make more inline w/ the docs (as you can have a webhook with no secret): https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#operation/createOneAlertConfigurationInOneProject
```suggestion
* `webhook_secret` - Optional authentication secret for the `WEBHOOK` notifications type.
```

Reply:
Updated